common/utils.cpp

```diff
@@ -390,6 +390,27 @@ PropertyValue DBusHandler::getDbusPropertyVariant(
     return value;
 }
 
+ObjectValueTree DBusHandler::getManagedObj(const char* service,
+                                           const char* rootPath)
+{
+    ObjectValueTree objects;
+    try
+    {
+        auto& bus = DBusHandler::getBus();
+        auto method = bus.new_method_call(service, rootPath,
+                                          "org.freedesktop.DBus.ObjectManager",
+                                          "GetManagedObjects");
+        auto reply = bus.call(method);
+        reply.read(objects);
+    }
+    catch (const std::exception& e)
+    {
+        std::cerr << "Look up of objects failed : " <<  service << std::endl;
+    }
+
+    return objects;
```

Comment:
is the dbus call fails ..what should be returned ?

a util function should not really have exception handling - its the function that calls this API that needs exception handling

Reply:
Handled the exception in caller function